packages/animated-pose/src/inc/default-transitions.ts

```diff
@@ -37,4 +37,7 @@ const intelligentTransition = {
   default: tween
 };
 
-export default new Map([['default', intelligentTransition]]);
+const defaultTransitions = new Map();
+defaultTransitions.set('default', intelligentTransition);
```

Comment:
`.set` returns the map itself and for tree-shakeability reasons it's generally better to keep single entities represented by a single expression, so I would advise doing this:
```js
// IE11 doesn't support `new Map(iterable)`, Map.prototype.set returns the map itself
export default new Map().set('default', intelligentTransition)
```

Reply:
Didn't realise you could chain `.set()` calls, nice! Amended.